src/utils/base64.ts

```diff
@@ -21,6 +22,8 @@ export function decodeHashFromBase64(hash: HoloHashB64): HoloHash {
  *
  * @public
  */
-export function encodeHashToBase64(hash: HoloHash): HoloHashB64 {
-  return `u${Base64.fromUint8Array(hash, true)}`;
+export function encodeHashToBase64(hash: Uint8Array): HoloHashB64 {
+  return String(new HoloHash( hash ));
 }
+
+export * from "@spartan-hc/holo-hash";
```

Comment:
This causes ambiguity because the `HoloHash` classes have the same name as all the `types.ts` exports.  How should we export the class implementations without conflict?

Reply:
I think we should ship this client with only one type for each type. So I wouldn't want to export a legacy type _and_ the new type under an alias. Therefore I'd replace the existing type with the spartan-hc one.

What do you think about this, @ThetaSinner?

Reply:
Supporting both `HoloHash` and `Uint8Array` input for most functions is important so that users are not forced to wrap all input in `new HoloHash( input )`.  But I don't know enough about typescript to do that.  Would it work to extend all the types with an "or-pipe"?

Example
```ts
export type AgentPubKey = Uint8Array | HoloHashLib.AgentPubKey;
```

Reply:
Nevermind, that doesn't solve the duplicate export issue.

Reply:
Just to let you know this is still on my radar.